pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -267,7 +255,6 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 			},
 			InitialDelaySeconds: 60,
 		},
-		Lifecycle:       opspec.PodLifeCycle(""),
```

Comment:
The lifecycle is replaced by the init container?

Reply:
Yes. Noah explains here why this has to be: https://github.com/rook/rook/pull/3594#discussion_r312279176